apps/frontend/src/components/proposal/ProposalTableOfficer.tsx

```diff
@@ -132,8 +132,15 @@ let columns: Column<ProposalViewData>[] = [
   },
   {
     title: 'PI Email',
-    field: 'principalInvestigator.email',
+    field: 'principalInvestigator',
     emptyValue: '-',
+    render: (proposalView) => {
+      if (proposalView.principalInvestigator?.email) {
+        return `${proposalView.principalInvestigator.email}`;
+      }
+
```

Comment:
We could use something likes this:  

Return proposalView.principalInvestigator.email ?? '';